test/final.test.js

```diff
@@ -24,13 +24,13 @@ test('throws if the supplied handler is not a function', async ({ throws }) => {
   }, Error('if supplied, the handler parameter should be a function'))
 })
 
-test('throws if not supplied logger with pino.destination or pino.extreme instance', async ({ throws, doesNotThrow }) => {
+test('throws if not supplied logger with pino.extreme or pino.extreme instance', async ({ throws, doesNotThrow }) => {
```

Comment:
```suggestion
test('throws if not supplied logger with pino.extreme instance', async ({ throws, doesNotThrow }) => {
```

Reply:
even though it's not necessary, can we add a smoke test to ensure that `pino.destination` it at least compatible with `pino.final`